analyzers/src/SonarAnalyzer.CSharp/Common/Walkers/CatchLoggingInvocationWalker.cs

```diff
@@ -30,17 +30,16 @@ namespace SonarAnalyzer.Common.Walkers;
 // - find all the logging invocations and check if the exception is logged
```

Comment:
The comment above does not apply to this walker anymore, since it doesn't save the re-throw.